sources/platform/actors/development/quick_start/start_locally.md

```diff
@@ -9,123 +9,144 @@ slug: /actors/development/quick-start/locally
 
 ---
 
-:::info Prerequisites
+## What you'll learn
 
-You need to have [Node.js](https://nodejs.org/en/) version 16 or higher with `npm` installed on your computer.
+- How to create and run your first Actor locally using Apify CLI
+- How to understand and configure Actor input, output, and storage
+- How to deploy your Actor to the Apify platform
 
-:::
+## Prerequisites
 
-## Install Apify CLI
+- [Node.js](https://nodejs.org/en/) version 16 or higher with `npm` installed on your computer.
+- The [Apify CLI](/cli/docs/installation) installed.
+- Optional: To deploy your Actor, [sign in](https://console.apify.com/sign-in).
 
-### MacOS/Linux
+## Step 1: Create your Actor
 
-You can install the Apify CLI via the [Homebrew package manager](https://brew.sh/).
+Use Apify CLI to create a new Actor:
 
 ```bash
-brew install apify-cli
+apify create
 ```
 
-### Other platforms
+The CLI will ask you to:
 
-Use [NPM](https://www.npmjs.com/) to install the Apify CLI.
-
-```bash
-npm -g install apify-cli
-```
+1. Name your Actor (e.g., `your-actor-name`)
+2. Choose a programming language (`JavaScript`, `TypeScript`, or `Python`)
+3. Select a development template
+   :::info
```

Comment:
```suggestion

   :::info
```

also the 3 spaces indent feels weird/inconsistent, i'd go with 4 spaces if that renders correctly (which it should, the very same file is using 4 spaces  on line 133)